upstream-community-operators/synopsys/synopsys-operator.2019.1.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,321 @@
+      createdAt: 2018-12-14T17:09:00Z
+      support: Synopsys
+      certified: "False"
+      alm-examples: '[{"apiVersion": "synopsys.com/v1", "kind": "OpsSight", "metadata": { "name": "opssight-test" }, "spec": { "namespace": "opssight-test", "perceptor": { "name": "opssight-core", "port": 3001, "image": "docker.io/blackducksoftware/opssight-core:2.2.2", "checkForStalledScansPauseHours": 999999, "stalledScanClientTimeoutHours": 999999, "modelMetricsPauseSeconds": 15, "unknownImagePauseMilliseconds": 15000, "clientTimeoutMilliseconds": 100000 }, "scannerPod": { "name": "opssight-scanner", "scanner": { "name": "opssight-scanner", "port": 3003, "image": "docker.io/blackducksoftware/opssight-scanner:2.2.2", "clientTimeoutSeconds": 600 }, "imageFacade": { "name": "opssight-image-getter", "port": 3004, "internalRegistries": [], "image": "docker.io/blackducksoftware/opssight-image-getter:2.2.2", "serviceAccount": "opssight-scanner", "imagePullerType": "skopeo" }, "replicaCount": 1 }, "perceiver": { "enableImagePerceiver": false, "enablePodPerceiver": true, "port": 3002, "imagePerceiver": { "name": "opssight-image-processor", "image": "docker.io/blackducksoftware/opssight-image-processor:2.2.2" }, "podPerceiver": { "name": "opssight-pod-processor", "image": "docker.io/blackducksoftware/opssight-pod-processor:2.2.2" }, "serviceAccount": "opssight-processor", "annotationIntervalSeconds": 30, "dumpIntervalMinutes": 30 }, "prometheus": { "name": "prometheus", "port": 9090, "image": "docker.io/prom/prometheus:v2.1.0" }, "enableSkyfire": false, "skyfire": { "image": "gcr.io/saas-hub-stg/blackducksoftware/pyfire:master", "name": "skyfire", "port": 3005, "prometheusPort": 3006, "serviceAccount": "skyfire", "hubClientTimeoutSeconds": 120, "hubDumpPauseSeconds": 240, "kubeDumpIntervalSeconds": 60, "perceptorDumpIntervalSeconds": 60 }, "enableMetrics": true, "defaultCPU": "300m", "defaultMem": "1300Mi", "logLevel": "debug", "secretName": "blackduck", "blackduck": { "user": "sysadmin", "initialCount": 0, "maxCount": 0, "concurrentScanLimit": 2, "totalScanLimit": 1000, "blackduckSpec": { "licenseKey": "byol-contact-synopsys", "persistentStorage": false, "certificateName": "default", "dbPrototype": "", "size": "small", "type": "worker", "imageRegistries": [ "docker.io/blackducksoftware/blackduck-authentication:2018.12.2", "docker.io/blackducksoftware/blackduck-documentation:2018.12.2", "docker.io/blackducksoftware/blackduck-jobrunner:2018.12.2", "docker.io/blackducksoftware/blackduck-registration:2018.12.2", "docker.io/blackducksoftware/blackduck-scan:2018.12.2", "docker.io/blackducksoftware/blackduck-webapp:2018.12.2", "docker.io/blackducksoftware/blackduck-cfssl:1.0.0", "docker.io/blackducksoftware/blackduck-logstash:1.0.2", "docker.io/blackducksoftware/blackduck-nginx:1.0.0", "docker.io/blackducksoftware/blackduck-solr:1.0.0", "docker.io/blackducksoftware/blackduck-zookeeper:1.0.0" ], "environs": [ "HUB_VERSION:2018.12.2" ] } } } },
```

Comment:
Is there an example for the Alerts CR? Or is this CR used internally by the operator?

Reply:
Good question.  It's an optional CR, and at the moment we are planning to not include it.  I'll remove the alerts.crd and commit to restart the CI tests.

Reply:
We added it back in, and an example exists now.